mta/tests/test_analysis_results_Test03.py

```diff
@@ -36,17 +36,11 @@ def test_analysis_results_search_sort_delete(application):
     analysis_results.search_analysis(row=2)
     view.clear_search()
     # Sort Analysis
+    before_sort_analysis_number = analysis_results.get_analysis_number(view, row=1)
     analysis_results.sort_analysis()
-    assert analysis_results.get_analysis_number(view, row=1) > analysis_results.get_analysis_number(
-        view, row=2
-    )
+    assert analysis_results.get_analysis_number(view, row=1) != before_sort_analysis_number
```

Comment:
This looks more appropriate assertion to me because the numbers of analysis will never be same , after sorting the number in first row will be greater than second row

      
     assert analysis_results.get_analysis_number(view, row=1) > analysis_results.get_analysis_number( view, row=2)

Reply:
It should be like this
```
 assert analysis_results.get_analysis_number(view, row=1) > analysis_results.get_analysis_number( view, row=2)
```